packages/perseus/src/perseus-types.ts

```diff
@@ -731,6 +736,23 @@ export type LockedPolygonType = {
     strokeStyle: "solid" | "dashed";
 };
 
+export type LockedFunctionType = {
+    type: "function";
+    color: LockedFigureColor;
+    strokeStyle: "solid" | "dashed";
+    equation: string; // This is the user-defined equation (as it was typed)
+    equationParsed?: {
```

Comment:
@mark-fitzgerald  The types in this file represent the "at rest" data representations for Perseus Data. As such, I try to avoid having keys that would never show up in Perseus JSON/data. 

As I read this PR, this key would be created the first time this locked figure was rendered and then "cached" here for future renders, right?  

My concern is folks thinking they can put data into this key in Perseus JSON and beginning to misuse the intent of this key (which I read as a memoization/cache of the parsed equation).

Reply:
Ah, thanks for that clarification. I wasn't aware of this. I will refactor this accordingly when I implement the add/edit capability.